sdk/src/main/scala/com/horizen/storage/SidechainStateStorage.scala

```diff
@@ -77,48 +81,129 @@ class SidechainStateStorage (storage : Storage, sidechainBoxesCompanion: Sidecha
     }
   }
 
-  def update(version : ByteArrayWrapper, withdrawalEpochInfo: WithdrawalEpochInfo,
-             boxUpdateList : Set[SidechainTypes#SCB],
-             boxIdsRemoveList : Set[Array[Byte]],
-             withdrawalRequestAppendList : Set[WithdrawalRequestBox]) : Try[SidechainStateStorage] = Try {
+  def getConsensusEpoch: Option[ConsensusEpochNumber] = {
```

Comment:
getConsensusEpoch -> getConsensusEpochNumber

is it possible to no have epochNumber at all?